docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -31,6 +31,10 @@ There you can find links to upgrade notes for other versions too.
     - [`Version20190121094400`](https://github.com/shopsys/shopsys/blob/master/packages/framework/src/Migrations/Version20190121094400.php <!--- TODO: change to released version instead of master -->
 
 ### Application
+- use configuration file to define directories that needs to be created during build of the application ([#781](https://github.com/shopsys/shopsys/pull/781))
+    - create new configuration file `app/config/directories.yml` with 2 types of directories `main_filesystem_directories` and `local_filesystem_directories`.
+    - add default directories according to these [changes](https://github.com/shopsys/shopsys/pull/781/files#diff-907bbe66c89cc8926cf1d44b5155e65a)
+    - if you had implemented your individual directories in `CreateApplicationDirectoriesCommand`, delete your extension of a class and add the directories into `directories.yml` and fill them into sections, you can read more about sections [here](https://github.com/shopsys/shopsys/blob/master/docs/intruduction/directories.yml) <!--- TODO: change to released version instead of master -->
```

Comment:
```suggestion
    - if you had implemented your individual directories in `CreateApplicationDirectoriesCommand`, delete your extension of a class and add the directories into `directories.yml` and fill them into sections. You can read more about sections [here](https://github.com/shopsys/shopsys/blob/master/docs/intruduction/directories.yml) <!--- TODO: change to released version instead of master -->
```